2017-02-08 14:54:18,005 - __main__ - logtest.py   ERROR : this is error
